sample-mpp-module/build.gradle

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Let's implement at least Android and iOS GUI in next PRs.
Otherwise this sample does not have any meaning.